bears/vcs/git/GitCommitBear.py

```diff
@@ -81,6 +98,16 @@ def run(self, allow_empty_commit_message: bool = False, **kwargs):
             stdout[1:],
             **self.get_body_checks_metadata().filter_parameters(kwargs))
 
+        stdout, _ = run_shell_command('git config --get-regex url')
```

Comment:
This should be in `check_issue` , and it is wasted computation if none of the relevant options are enabled.